static/src/javascripts/projects/common/modules/experiments/tests/integrate-criteo.ts

```diff
@@ -0,0 +1,21 @@
+import type { ABTest } from '@guardian/ab-core';
+import { noop } from '../../../../../lib/noop';
+
+export const integrateCriteo: ABTest = {
+	id: 'IntegrateCriteo',
+	author: 'Chris Jones (@chrislomaxjones)',
+	start: '2021-11-22',
+	expiry: '2022-01-10',
+	audience: 2 / 100,
+	audienceOffset: 0,
```

Comment:
This is where you’d want to use an offset:

```suggestion
	audienceOffset: 96 / 100,
```